src/app/shared/ui/circular-button/circular-button.scss

```diff
@@ -0,0 +1,80 @@
+:host button{
```

Comment:
sprwadź proszę czy prettier formatuje ci twoje pliki `scss`

Reply:
Wygląda na to ze nie do końca, staram się to ogarnąć

Reply:
Udalo się naprawić dzialanie prettiera